quiz/equal-iv_en.md

```diff
@@ -1,4 +1,34 @@
+> This awesome solution is created by [@abhijeet-saxena](https://github.com/abhijeet-saxena)
 
-There is no solution yet.
+## Problem
 
-Would you like to [contribute to the solution](https://github.com/BFEdev/BFE.dev-solutions/blob/main/quiz/equal-iv_en.md)? [Contribute guideline](https://github.com/BFEdev/BFE.dev-solutions#how-to-contribute)
+```js
+console.log([0] == "");
+console.log([0] == 0);
+```
+
+## Solution
+
+```
+false
+true
+```
+
+## Explanation
+
+Here, in both cases the left operand `[0]` is an `object` while the right operand is a [primitive](https://developer.mozilla.org/en-US/docs/Glossary/Primitive) (string in first line and number in the second)
+
+The [equality (==) operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Equality) checks whether its two operands are equal, and unlike the strict equality operator ===, it attempts to convert and compare operands that are of different types.  To do this comparison, JS will follow the rules for [Loose equality using ==](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness#loose_equality_using) which states that:
```

Comment:
MDN is spec explainer, would you mind switch the MDN links to ECMA spec links ? like this https://tc39.es/ecma262/multipage/abstract-operations.html#sec-islooselyequal